cfme/common/vm.py

```diff
@@ -446,33 +446,69 @@ def _refresh():
             self.load_details()
 
     def set_ownership(self, user=None, group=None, click_cancel=False, click_reset=False):
-        """Set ownership of the VM/Instance or Template/Image"""
-        view = navigate_to(self, "SetOwnership")
-        view.form.fill({'user_name': user, 'group_name': group})
+        """Set instance ownership
+
+        Args:
+            user (str): username for ownership
+            group (str): groupname for ownership
+            click_cancel (bool): Whether to cancel form submission
+            click_reset (bool): Whether to reset form after filling
+        """
+        view = navigate_to(self, 'SetOwnership')
+        fill_result = view.form.fill({
+            'user_name': user,
+            'group_name': group})
+        if not fill_result:
+            view.flash.assert_no_error()
+            view.form.cancel_button.click()
+            view = self.create_view(navigator.get_class(self, 'Details').VIEW)
+            view.flash.assert_success_message('Set Ownership was cancelled by the user')
```

Comment:
**REQUIRED**
Here and further. We shouldn't assert flash messages inside the framework. Just keep `view.flash.assert_no_error()`. If you want to assert the flash message you can create a separate test case for that.

Reply:
@quarckster thoughts on leaving this as-is since this is moving the method into the common classes, and addressing our flash message move in a separate PR? Otherwise she'll need to write the test against this message as well for this PR to be complete.